### PR TITLE
Edits

### PR DESCRIPTION
**Pull request by misscs**
Thoughts and ideas. Opened as a PR to facilitate discussion and to allow original author to retain editorial say over the post

### CODE REVIEW

File: source/blog/2015-07-15-submission-stats.markdown [hunk 1]
```diff
@@ -14,52 +12,55 @@ We had 81 total submissions in 2015!
 
 <em><small>The above numbers are total submissions; about 14-18 submissions were tagged in 2 categories.</small></em>
 
-If workshops are 3 hours, talks 1, discussions 1, and lightning talks .25, that’s a grand total of **83 hours of content**! And while we seriously debated making SassConf two weeks long, in the end, we couldn’t pick everything. (Sad trombone.)
+If workshops are 3 hours, talks 1 hour, discussions 1 hour, and lightning talks 15 minutes, that's a grand total of **83 hours of content**! We  seriously debated making SassConf two weeks long, in the end we couldn’t pick everything. (Sad trombone.)
```
Comment (misscs) [inline source/blog/2015-07-15-submission-stats.markdown:15]: @elyseholladay I was thinking an animated gif that conveys _sad trombone_ could be rad

File: source/blog/2015-07-15-submission-stats.markdown [hunk 2]
```diff
@@ -14,52 +12,55 @@ We had 81 total submissions in 2015!
...
+Of the total 81 submissions in 2015, **67** were from unique submitters. Of those.
 
-* **67** unique submitters. Of those,
 * **48** (72%) were men
-* **19** (28%) were women. 
+* **19** (28%) were women.
 
 > Submissions from women went up to 28%<br/> from last year's 5%!
 
-Thanks, ladies, for making us look so good!
```
Comment (misscs) [inline source/blog/2015-07-15-submission-stats.markdown:43]: can we convey this with an animated gif?

File: source/blog/2015-07-15-submission-stats.markdown [hunk 3]
```diff
@@ -115,29 +116,41 @@ I was also surprised how many varied locations our awesome submitters are from!
...
+It was a relatively small, but a very rewarding success.
 
-All but 4 submitted their talks. We don't know why they chose not to submit.
+* 13 people asked (and received) CFP mentoring office hours;
+* all but 4 submitted their talks
 
 > Overall, CFP mentorship was great, and I’m really psyched we had the program. I had some great moments with Adekunle and it was awesome to see him go on to CSS Conf with the talk we worked out. Nice to see our resources help not just make great talks for SassConf but also empower the Sass community to speak elsewhere. — Scott Kellum
 
-Overall, we had a very successful CFP and the overall quality of submissions was fantastic. I know a few of you had concerns about the open CFP on GitHub, but we didn't get any email/private submissions.
+s### Reflections on the open CFP process
```
Comment (misscs) [inline source/blog/2015-07-15-submission-stats.markdown:128]: Whoops remove typo! remove the 's'

File: source/blog/2015-07-15-submission-stats.markdown (hunk 1 above)
Comment (scottkellum) [inline source/blog/2015-07-15-submission-stats.markdown:15]: @misscs @elyseholladay Maybe this gif? Too much content

File: source/blog/2015-07-15-submission-stats.markdown (hunk 2 above)
Comment (scottkellum) [inline source/blog/2015-07-15-submission-stats.markdown:43]: http://media.giphy.com/media/jlNmaotN2GEec/giphy.gif

File: source/blog/2015-07-15-submission-stats.markdown (hunk 1 above)
Comment (elyseholladay) [inline source/blog/2015-07-15-submission-stats.markdown:15]: Ha, love it. I'll find a good one!

File: source/blog/2015-07-15-submission-stats.markdown [hunk 4]
```diff
@@ -14,52 +12,55 @@ We had 81 total submissions in 2015!
...
+We gleaned some other demographic data about our submitters, as well.
 
-* Black Men: **2** 
+* Black Men: **2**
 * Black Women: **1**
 * Asian Men: **4**
 * Asian Women: **1**
-* Latina: **2**
-* LGBTQ+: **5**
+* Latin@: **2**
```
Comment (elyseholladay) [inline source/blog/2015-07-15-submission-stats.markdown:52]: @misscs actually, both our submissions were from Latinas, specifically, so the specification was intentional :)

File: source/blog/2015-07-15-submission-stats.markdown [hunk 5]
```diff
@@ -14,52 +12,55 @@ We had 81 total submissions in 2015!
...
+* LGBTQIA: **5**
```
Comment (elyseholladay) [inline source/blog/2015-07-15-submission-stats.markdown:53]: @misscs Thanks—I think I changed this elsewhere and forgot to come back for this one!

File: source/blog/2015-07-15-submission-stats.markdown (hunk 3 above)
Comment (elyseholladay) [inline source/blog/2015-07-15-submission-stats.markdown:128]: :+1:

File: source/blog/2015-07-15-submission-stats.markdown [hunk 6]
```diff
@@ -115,29 +116,41 @@ I was also surprised how many varied locations our awesome submitters are from!
...
+Firstly, and most glaringly, there is still a tremendous lack of submissions from people of color. For a conference with quite a few people of color on its organizing team, we failed to see any improvement in this area.
 
-#### Where can we improve?
+The only way to increase the possibility of speakers of color is to increase the number of submissions. And the only way to increase the number of submissions is to directly encourage and ensure our poc members are submitting. We believe this starts at the local level.
 
-Overall, I feel we did well. That said, there are a handful of places we can improve upon for next year.
+As I mentioned before, I should have done more outreach. Karolina tweets about the work it takes to ensure a diverse event, and she isn't lying.
 
-Firstly, and most glaringly, is the lack of submissions from people of color. For a conference with quite a few people of color on its organizing team, we dropped the ball here. The only way to increase the possibility of speakers of color is to increase the number of submissions.
+_<insert tweet here>_
```
Comment (elyseholladay) [inline source/blog/2015-07-15-submission-stats.markdown:147]: ^ totally meant to do that and forgot, thanks!

**Comment by elyseholladay** [comment]
merged to redesign branch, closing

File: source/blog/2015-07-15-submission-stats.markdown (hunk 4 above)
Comment (misscs) [inline source/blog/2015-07-15-submission-stats.markdown:52]: Ah, I was just being gender neutral since we haven't specially asked their PGP

**Comment by misscs** [comment]
I'm sending the revised version off to a few folks. I've told them we need input ASAP. Will open another PR with suggestions.

**Pull request closed without merging**